pipemgr/src/osbook_utils.py

```diff
@@ -6,15 +6,16 @@
 
 OSBOOKS_FILE = OUTPUT_ROOT/"osbooks.yml"
 
+
 class OSBook:
     def __init__(self, book_repo: str, archive_server: str):
         self.book_repo = book_repo
         self.archive_server = archive_server
 
     def __eq__(self, o: object) -> bool:
         return (isinstance(o, OSBook)
-            and o.book_repo == self.book_repo
-            and o.archive_server == self.archive_server)
+                and o.book_repo == self.book_repo
```

Comment:
I take it the extra indent is intentional?

Reply:
That and many other whitespace changes were added by automatic formatting. It's within parenthesis too, so indent doesn't really matter.